test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1020,41 +1162,46 @@ def test_cgroup_cpuset(self):
         self.server.status(JOB, attrib, jid2)
         filename2 = j2.attributes[ATTR_o]
         ehost2 = j2.attributes['exec_host']
-        self.logger.info("Job1 .o file: %s" % filename1)
+        self.logger.info('Job1 .o file: %s' % filename1)
         tmp_file1 = filename1.split(':')[1]
         tmp_host1 = ehost1.split('/')[0]
-        tmp_out1 = self.wait_and_read_file(filepath=tmp_file1, mom=tmp_host1)
-        self.logger.info("tmp_out1: %s" % tmp_out1)
+        tmp_out1 = self.wait_and_read_file(filename=tmp_file1, mom=tmp_host1)
+        self.logger.info('tmp_out1: %s' % tmp_out1)
         self.wait_and_remove_file(filename=tmp_file1, mom=tmp_host1)
         self.assertTrue(
             jid1 in tmp_out1, '%s not found in output on host %s'
             % (jid1, tmp_host1))
-        self.logger.info("Job2 .o file: %s" % filename2)
+        self.logger.info('Job2 .o file: %s' % filename2)
         tmp_file2 = filename2.split(':')[1]
         tmp_host2 = ehost2.split('/')[0]
-        tmp_out2 = self.wait_and_read_file(filepath=tmp_file2, mom=tmp_host2)
-        self.logger.info("tmp_out2: %s" % tmp_out2)
+        tmp_out2 = self.wait_and_read_file(filename=tmp_file2, mom=tmp_host2)
+        self.logger.info('tmp_out2: %s' % tmp_out2)
         self.wait_and_remove_file(filename=tmp_file2, mom=tmp_host2)
         self.assertTrue(
             jid2 in tmp_out2, '%s not found in output on host %s'
             % (jid2, tmp_host2))
-        self.logger.info("job dir check passed")
-        if 'CpuIDs=0' in tmp_out1 and 'CpuIDs=1' in tmp_out2:
-            pass
-        elif 'CpuIDs=1' in tmp_out1 and 'CpuIDs=0' in tmp_out2:
-            pass
-        else:
-            self.assertTrue(False,
-                            "Processes should be assigned to different CPUs")
-        self.logger.info("CpuIDs check passed")
+        self.logger.info('job dir check passed')
+        for kv in tmp_out1:
+            if 'CpuIDs=' in kv:
+                cpuid1 = kv
+                break
+        for kv in tmp_out2:
+            if 'CpuIDs=' in kv:
+                cpuid2 = kv
+                break
+        self.logger.info('cpuid1 = ' + cpuid1)
+        self.logger.info('cpuid2 = ' + cpuid2)
```

Comment:
Are these two needed?  You will get a log message that the check failed or passed.

Reply:
No. Just more test debugging messages.